common/src/main/java/com/amazonaws/services/schemaregistry/common/SchemaByDefinitionFetcher.java

```diff
@@ -10,9 +10,9 @@
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 import lombok.SneakyThrows;
+import software.amazon.awssdk.services.glue.model.Compatibility;
 
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Same as above on imports.

Reply:
Restored back to original with no changes